bin/main.rb

```diff
@@ -1,19 +1,20 @@
 #!/usr/bin/env ruby
 
-# rubocop:disable Lint/MissingCopEnableDirective, Style/GuardClause, Metrics/MethodLength, Style/MixinUsage
+# rubocop:disable Style/MixinUsage, Lint/MissingCopEnableDirective, Style/GuardClause
 
 # require 'colorize'  # Need to run 'gem install colorize'
 
 # self customized libary
 
 module Interface
   def show(positions)
+    puts
     puts "\t  #{positions[6]} | #{positions[7]} | #{positions[8]}"
     puts "\t ---|---|---"
     puts "\t  #{positions[3]} | #{positions[4]} | #{positions[5]}"
     puts "\t ---|---|---"
-    puts "\t  #{positions[0]} | #{positions[1]} | #{positions[2]}"
-    puts ''
+    puts "\t  #{positions[0]} | #{positions[1]} | #{positions[2]}"
+    puts
   end
```

Comment:
[OPTIONAL]

- [ ] DRY rule should be respected. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/dry.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- I think the code in your show method is repetitive and  could be done with a loop of some sort, please consider changing it.